### PR TITLE
cTab settings module.

### PR DESCRIPTION
**Pull request by EM-Creations**
**When merged this pull request will:**
- Adds a new cTab settings module.
- I have tested this and it appears to work.

Requested here: https://github.com/Global-Conflicts-ArmA/Olsen-Framework-Arma-3/issues/155

### CODE REVIEW

File: modules/cTab_settings/functions/CTAB_SET/script_component.hpp [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#define COMPONENT CAPZONE
```
Comment (PiZZAD0X) [inline modules/cTab_settings/functions/CTAB_SET/script_component.hpp:1]: Incorrect component name, no functions are used so functions can be taken out couldn't they?

**Review by PiZZAD0X — CHANGES_REQUESTED** [review]
cTab also uses the postInit handler to set its setting variables. Potentially means that cTab init can come after this init. Recommend we have a cba waitUntilAndExecute with a cTab set variable as a condition, then change the cTab variables.

File: modules/cTab_settings/functions/CTAB_SET/script_component.hpp (hunk 1 above)
Comment (PiZZAD0X) [reply modules/cTab_settings/functions/CTAB_SET/script_component.hpp]: Yeah, easy enough to add in when it's needed.

**Comment by EM-Creations** [comment]
> 

That's a good solution, but would that cause any problems if the mod was removed? Would CBA waiting for something that never happens ever be a problem or a big performance issue?

**Comment by PiZZAD0X** [comment]
> 

wouldn't be a big performance issue but you are right, needs the cfgPatches check. Pushed the exit in the last commit.

**Review by PiZZAD0X — APPROVED** [review]
Happy with this

**Pull request merged**